WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java

```diff
@@ -80,6 +77,12 @@ public void wPScreenshotTest() {
     private void editBlogPost() {
         (new MySitesPage()).switchToSite("fourpawsdoggrooming.wordpress.com")
                            .goToPosts();
+
+        // There is a possibility of the edit post getting stuck with an `AppNotIdleException`
+        // On the UI it's shown by the flashing progress indicator at the bottom. This idle
+        // appears to wait long enough for the edit screen to show properly
+        idleFor(3000);
```

Comment:
Great finding, good thing I wasn't spending more time on it 😅 , I was on a totally different path that could have taken much more time.